crates/virtual-node/src/lib.rs

```diff
@@ -107,8 +107,8 @@ impl VirtualNode {
     ///
     /// let div = VirtualNode::text("div");
     /// ```
-    pub fn text(text: &str) -> Self {
-        VirtualNode::Text(text.into())
+    pub fn text<S>(text: S) -> Self where S: Into<String> {
+        VirtualNode::Text(VElement::new(text.into()))
```

Comment:
oh sorry I merged to soon.. looks like this doesn't compile

should this be `VirtualNode::text(text.into())` ?

Reply:
Huh, I fixed that... Something must have gone wrong when force-pushing. Sorry for that.

For some reason the double-into does not work without type hints, so if you simply change it to `VirtualNode::Text(VText::new(text.into()))` it'll work!

Reply:
Hmmm still failing https://travis-ci.org/chinedufn/percy/jobs/493356098

Did tests pass for you locally with that?

Here's my commit https://github.com/chinedufn/percy/commit/424fb947d3ef0d50c6c32dcfe567bcfa230e48c6

I committed it from GitHub but I can take a look tonight by running tests locally unless you already know the issue

Reply:
Oh, I overlooked an example. Sorry! Will fix it.

Reply:
I hope https://github.com/chinedufn/percy/commit/d117d96a9649204c2a8a90b140e0963e6f10a21a fixes the issue.

`&String` does not implement `Into<String>`, so we either need to deref (`&*mystring`) or move/clone the string.

Reply:
I also had two other test failures, but I think they're unrelated. Something about multiple versions of `virtual_dom_rs`, could be a local issue.

Reply:
Looks like tests are passing, thanks!!